docs/import/manage_layers.js

```diff
@@ -312,37 +306,15 @@ function populateStateAndDisasterAssetPickers(disaster) {
   const assetPickerDiv = $('.asset-pickers');
   assetPickerDiv.empty();
 
-  const promises = [];
   if (disasterAssets.has(disaster)) {
     createDisasterAssetPicker(disaster);
   } else {
     const disasterDone = getDisasterAssetsFromEe(disaster).then((assets) => {
       disasterAssets.set(disaster, assets);
       createDisasterAssetPicker(disaster);
     });
-    promises.push(disasterDone);
+    return disasterDone;
   }
-
-  const states = getCurrentData()['states'];
-  const statesToFetch = [];
-  for (const state of states) {
-    if (!stateAssets.has(state)) statesToFetch.push(state);
-  }
-  // TODO: add functionality to re-pull all cached states from ee without
-  // reloading the page.
-  if (statesToFetch.length === 0) {
-    createStateAssetPickers(states);
-  } else {
-    const statesDone = getStatesAssetsFromEe(statesToFetch).then((assets) => {
-      for (const asset of assets) {
-        stateAssets.set(asset[0], asset[1]);
-      }
-      createStateAssetPickers(states);
-    });
-    promises.push(statesDone);
-  }
-
-  return Promise.all(promises);
 }
```

Comment:
Sorry if my other comments were confusing, I think you can delete createStateAssetPickers and its export!